core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -238,12 +238,34 @@ class Main {
                     info("TCP timeouts: $timeouts")
                 }
 
+                // The number of covered targets and the number of potential faults will be shown in both
+                // blackbox and whitebox
+                val elementsPair = data.find{it -> it.header == "coveredTargets"}
```

Comment:
never refer to strings like this, as it is very brittle. rather introduce a constant in `Statistics` (eg see ones in the companion object).
anyway, this `coveredTarget` is not necessary here

Reply:
Thank you so much Andrea for your comment. I have removed that since from my understanding, "coveredTargets" should be part of the white-box testing, not the black-box testing.